### ISSUE TITLE
Bibliography in search results

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#2094]
A remark/question:
I relatively often notice that among search results there are records in which the term I searched for is part of the title of a bibliographic reference. 
For example, if I search for "Iyasu" and limit to persons, I get every person record in which Guidi, I. 1903. Annales Iohannis I, Iyāsu I, Bakāffā: Textus is mentioned, such as for example https://betamasaheft.eu/persons/PRS3430DawitII/main, which contains no other mention of Iyasu.
These records are often irrelevant to my search and inflate the list of search results. 
My impression would be that it could be helpful if the search would ignore bibliography. However, there probably are good reasons why it does not - it could maybe help my frustration in these cases if I knew these reasons. Could you remind me of them? Or, if there are none, did you also experience this as problematic?
@eu-genia @DenisNosnitsin1970

**Comment by eu-genia** [comment]
I agree would be good to ignore; I have not looked into search encoding though, I don't think for me at least it is worth the effort, but we can put it on the wishlist...

**Comment by DenisNosnitsin1970** [comment]
I agree, there is no need that terms/names are looked for in bibliography.

**Comment by eu-genia** [comment]
since bibliography titles are stored in the expanded xml they are clearly searchable. the context of excluding terms if they appear inside of bibl should be added to full search (no way to exclude from simple search though which searches everywhere)
